taskcluster/ci/toolchain/android.yml

```diff
@@ -7,6 +7,19 @@ job-defaults:
     worker:
         max-run-time: 1800
 
+linux64-android-gradle-repack:
```

Comment:
Gradle and the SDK could have been generated as a single TC task, but taskgraph assumes toolchain just output a single artifact. Moreover, this makes sure the gradle home doesn't contain any build dependencies.

Reply:
Looking at m-c, it appears that it has `gradle-dist` as another top-level directory, so we could get by with a single dependency. That said, this likely changes less often, so decoupling them is probably a reasonable thing to do. It might make sense to suggest and/or implement this split for m-c too.